06_timers/kernel_space/time.c

```diff
@@ -0,0 +1,71 @@
+// SPDX-License-Identifier: GPL-2.0
+#include <asm/uaccess.h>
```

Comment:
check patch WARNING: Use #include <linux/uaccess.h> instead of <asm/uaccess.h>
Additional question - are you sure all these includes needed here?